tests/chainer_tests/links_tests/normalization_tests/test_decorrelated_batch_normalization.py

```diff
@@ -73,68 +70,107 @@ def _calc_mean(x, groups):
     # NOTE(crcrpar): np.linalg.eigh does not support float16
     'dtype': [numpy.float32, numpy.float64],
 })))
-class DecorrelatedBatchNormalizationTest(unittest.TestCase):
+@testing.inject_backend_tests(
+    None,
+    # CPU tests
+    [{}]
+    # GPU tests
+    + testing.product({
+        'use_cuda': [True],
+        'use_cudnn': ['never', 'always'],
```

Comment:
We don't have to parameterize over cuDNN because it is not used.

Reply:
Fixed!